### PR TITLE
Make a queries for multiples tables

### PR DESCRIPTION
**Pull request by mgmediaweb**
- [x] What animals belong to Melody Pond?
image

- [x] List of all animals that are pokemon (their type is Pokemon).
image
 
- [x] List all owners and their animals, remember to include those that don't own any animal.
image

- [x] How many animals are there per species?
image

- [x] List all Digimon owned by Jennifer Orwell.
image

- [x] List all animals owned by Dean Winchester that haven't tried to escape.
image

- [x] Who owns the most animals?
image

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -10,3 +10,24 @@ CREATE TABLE animals (
...
+    id SERIAL NOT NULL,
+    name VARCHAR(100) NOT NULL,
+    PRIMARY KEY (id)
+);
+
+ALTER TABLE animals 
+    DROP COLUMN species;
+
+ALTER TABLE animals
+    ADD COLUMN owner_id INT REFERENCES owners(id);
```
Comment (ShahierNashaat) [inline schema.sql:36]: - [x] Kindly be noticed that you should add the `species_id` to the `animals` table.
```sql
ALTER TABLE animals ADD species_id int REFERENCES species(id);
```

File: queries.sql [hunk 2]
```diff
@@ -106,3 +106,94 @@ SELECT species, AVG(escape_attempts) as AVERAGE_ESCAPES
...
+
+/* UPDATE ESPECIES ID */
+
+UPDATE animals
+	SET species_id=1
+	WHERE name LIKE '%mon';
+
+UPDATE animals
+	SET species_id=2
+	WHERE name NOT LIKE '%mon';
```
Comment (ShahierNashaat) [inline queries.sql]: - [x] Kindly be noticed that you are assigning the `Pokemon` species to the animals with the name end with `mon`, but it should be `Digimon` species to the animals with the name end with `mon`.
- [x] Also notice that you should put the update commands in the `data.sql` file.
```sql
   UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Digimon')
   WHERE name like '%mon';

   UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Pokemon')
   WHERE species_id IS NULL;
```

File: queries.sql [hunk 3]
```diff
@@ -106,3 +106,94 @@ SELECT species, AVG(escape_attempts) as AVERAGE_ESCAPES
...
+
+UPDATE animals
+	SET owner_id=4
+	WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
+
+UPDATE animals
+	SET owner_id=5
+	WHERE name IN ('Angemon', 'Boarmon');
+
+/* What animals belong to Melody Pond */
```
Comment (ShahierNashaat) [inline queries.sql]: - [x] Also notice that you should put the `update` commands in the `data.sql` file.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @mgmediaweb, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 3 above)
Comment (mgmediaweb) [reply queries.sql]: done!

File: queries.sql (hunk 2 above)
Comment (mgmediaweb) [reply queries.sql]: done!

File: schema.sql (hunk 1 above)
Comment (mgmediaweb) [reply schema.sql:36]: done!

**Review by Lordkaito — APPROVED** [review]
Hi @mgmediaweb, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**